src/Schema/ScreenOwnerType.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * All valid screen (page) types in Eigen
```

Comment:
I wonder if these are specific to Eigen, do we have an equivalent in force around page owner?

Reply:
Some of them are specific to Eigen (city guide screens, for example), but there will be a lot of overlap. I would imagine if we implement something similar to this event for Force (i.e. `clickedEntityGroup`), we could use the same list. The main difficulty here is a naming one: we have pages and clicks for Force and screens and taps for Eigen.

Reply:
We can still share the owner type, and also create a second layer to limit by different contexts. Sharing this field, if there is significant overlap, will not overlap with the clicks/taps. We do something similar with `AuthContextModule`, which exposes a limited set of `ContextModule` values for use in authentication components.

Reply:
@eessex something that might be helpful for both Eigen and Force is requiring these three properties on all events: `context_screen_owner_type`, `context_screen_owner_slug`, `context_screen_owner_id`. We've discussed this before but I don't think we ever came to a final decision!

These properties will tell us what specific page (like artwork or artist) an event occurred on. We already get a `path` and `url` from Segment's automated [common fields](https://segment.com/docs/connections/spec/common/), but this extra data will help us do more focused analysis. Lmk if you think this could be automated for all events!

Reply:
@abhitip the only reason i left `slug` and `id` optional was because of the 'Home' screen, which doesn't have either. I can see `type` being required all the time, but there might be general screens that don't have "roots."

Reply:
Yup makes sense! I have a feeling we can still automate this for all events - either we always pass all 3 properties but allow for 2 of them to be NULL or we have some if/else statement to handle it. I'll discuss it with eve!